index.html

```diff
@@ -554,26 +554,38 @@ <h4>The MediaStreamTrackAudioStats interface</h4>
           and
           <dfn class=export data-dfn-for="MediaStreamTrackAudioStats">[[\DroppedFramesDuration]]</dfn>,
           initialized to 0.</p>
+          <p>Let the {{MediaStreamTrackAudioStats}} also have an internal slot
+          <dfn class=export data-dfn-for="MediaStreamTrackAudioStats">[[\CurrentTaskId]]</dfn>
+          initialized to <code>null</code>.</p>
           <p>The <dfn data-lt="expose audio frame counters steps">expose audio
           frame counters steps</dfn> are the following:</p>
           <ul>
             <li>
-              <p>If this is the first time the [= expose audio frame counters
-              steps =] are called for this {{MediaStreamTrackAudioStats}} in
-              this task execution cycle, set the internal slots of as follows:
-              {{MediaStreamTrackAudioStats/[[DeliveredFrames]]}} is set to
+              <p>Let <var>taskId</var> be a number unique to the current
+              task.</p>
+            </li>
+            <li>
+              <p>If {{MediaStreamTrackAudioStats/[[CurrentTaskId]]}} is equal to
+              <var>taskId</var>, abort these steps.</p>
```

Comment:
The alternative, if we don't want to invent new concepts, is to say:

Queue a microtask to set `[[NeedsUpdate]]` to `true`.

And to set it to `false` when we update the counters.

The implementation could still use a task ID internally since the behavior would be identical, but our spec wouldn't need to invent new concepts (I don't think task IDs is an existing concept).